src/Service/PaymentStateDataService.php

```diff
@@ -26,23 +26,27 @@
 
 use Adyen\AdyenException;
 use Adyen\Shopware\Entity\PaymentStateData\PaymentStateDataEntity;
+use JetBrains\PhpStorm\ArrayShape;
```

Comment:
`JetBrains\PhpStorm\ArrayShape` this import and related type declaration can only be used on PhpStorm and might create compatibility issues on other IDEs.